build-setup-nolog.sh

```diff
@@ -64,6 +69,21 @@ do
     shift
 done
 
+# before doing anything verify that you are on a release branch/tag
+set +e
+tag=$(git describe --exact-match --tags)
+tag_ret_code=$?
+set -e
+if [ $tag_ret_code -ne 0 ]; then
+    if [ "$SKIP_VALIDATE" = false ]; then
+        read -p "WARNING: You are not on a tagged release of FireSim.\nType \"ok\" to continue if this is intended, otherwise see https://docs.fires.im/en/stable/Initial-Setup/Setting-up-your-Manager-Instance.html#setting-up-the-firesim-repo: " validate
+        [[ $validate == [oO][kK] ]] || exit 5
+        echo "Setting up non-release FireSim"
```

Comment:
Recommand making this wait for the user to enter `y` or `Y`, which is more unix-y.